pkgs/top-level/all-packages.nix

```
@@ -7371,6 +7371,9 @@ in
 
   colm = callPackage ../development/compilers/colm { };
 
+  colmap = callPackage ../applications/science/misc/colmap { };
+  colmapWithCuda = colmap.override { cudaSupport = true; };
```

```suggestion
  colmapWithCuda = callPackage ../application/science/misc/colmap { cudaSupport = true; };
```
I'm not actually sure which is preferred so let someone else comment before changing this

---

if it's short, putting it on a single line is fine. if it's longer typically use:
```
pkgs.callPackages ../path/to {
  foo = true;
  bar = false;
};